code/modules/projectiles/ammo_types/energy_ammo.dm

```diff
@@ -739,3 +739,35 @@
 
 /datum/ammo/energy/plasma_pistol/do_at_max_range(turf/T, obj/projectile/proj)
 	drop_fire(T, proj)
+
+/datum/ammo/energy/particle_lance
+	name = "particle beam"
+	hitscan_effect_icon = "lance"
+	hud_state = "plasma_blast"
+	hud_state_empty = "battery_empty_flash"
+	ammo_behavior_flags = AMMO_ENERGY|AMMO_HITSCAN|AMMO_PASS_THROUGH_MOB
+	bullet_color = COLOR_DISABLER_BLUE
+	armor_type = ENERGY
+	max_range = 21
+	accurate_range = 15
+	accuracy = 5
+
```

Comment:
```suggestion
	accuracy = 5
```

Reply:
I assume this was meant to remove the space, done.